<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LT8304" failures="3" tests="6" name="Test" time="14.237" errors="0" timestamp="2025-07-08T17:51:45 IST">
  <testcase classname="api.test.UserDataDrivenTests" name="testPostUser" time="4.443"/>
  <testcase classname="api.test.UserDataDrivenTests" name="testPostUser" time="1.58"/>
  <testcase classname="api.test.UserDataDrivenTests" name="testPostUser" time="1.575"/>
  <testcase classname="api.test.UserDataDrivenTests" name="testDeleteUser" time="1.575">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;404&amp;gt; but was:&amp;lt;200&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<404> but was:<200>
at api.test.UserDataDrivenTests.testDeleteUser(UserDataDrivenTests.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUser -->
  <testcase classname="api.test.UserDataDrivenTests" name="testDeleteUser" time="1.346">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;404&amp;gt; but was:&amp;lt;200&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<404> but was:<200>
at api.test.UserDataDrivenTests.testDeleteUser(UserDataDrivenTests.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUser -->
  <testcase classname="api.test.UserDataDrivenTests" name="testDeleteUser" time="1.451">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;404&amp;gt; but was:&amp;lt;200&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<404> but was:<200>
at api.test.UserDataDrivenTests.testDeleteUser(UserDataDrivenTests.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUser -->
</testsuite> <!-- Test -->
